test/unit/constraints/string.test.ts

```diff
@@ -464,7 +450,7 @@ describe('String constraints', () => {
         testConstraint<string>(
             () => {
                 class MyClass {
-                    @Regex(/test/g)
+                    @Regex(/test/)
```

Comment:
Joi's regex validator doesn't like the regex having the global flag. Joi seems to treat it like it is global regardless.